fe/fe-core/src/main/java/com/starrocks/server/CatalogMgr.java

```diff
@@ -199,6 +204,10 @@ private static class SerializeData {
 
     }
 
+    public ConcurrentHashMap<String, Catalog> getCatalogs() {
```

Comment:
is it only used for UT, could you mock `createCatalog` in UT instead.

Reply:
+1, you should delete this function.

Reply:
done